client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -126,7 +126,7 @@ class CustomContentTypes extends Component {
 
 	renderTestimonialSettings() {
 		const { translate } = this.props;
-		const fieldLabel = translate( 'Testimonials' );
+		const fieldLabel = translate( 'testimonials' );
```

Comment:
:disappointed: 47 existing translations will be lost with this change.
If the string must change, the following option has already been translated **32** times:
`translate( 'Testimonial' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/lowercase-using-css-for-translation-object)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).